sql/src/main/java/org/apache/druid/sql/avatica/DruidMeta.java

```diff
@@ -139,7 +147,9 @@ public DruidMeta(
                 .setDaemon(true)
                 .build()
         ),
-        authMapper.getAuthenticatorChain()
+        authMapper.getAuthenticatorChain(),
+        // To prevent server hammering, the timeout must be at least 1 second.
+        new ResultFetcherFactory(Math.max(1000, config.getFetchTimeoutMs()))
```

Comment:
No need to call `Math.max` because the ctor of  `ResultFetcherFactory` already guarantees the minimal timeout is 1 sec.